pandas/core/arrays/base.py

```diff
@@ -992,6 +995,35 @@ def _ndarray_values(self) -> np.ndarray:
         """
         return np.array(self)
 
+    def _accumulate(self, name, skipna=True, **kwargs):
+        """
+        Return an array result of performing the accumulation operation.
+
+        Parameters
+        ----------
+        name : str
+            Name of the function, supported values are:
+            { cummin, cummax, cumsum, cumprod }.
+        skipna : bool, default True
+            If True, skip NaN values.
+        **kwargs
+            Additional keyword arguments passed to the accumulation function.
+            Currently, no is the only supported kwarg.
```

Comment:
What's "no"?

Reply:
just intended to show my insecurity here, if we need it.